charts/Pie.args.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import Viz from "../primitives/Viz";
+
+import {Pie as D3plusPie} from "d3plus-react";
+export const Pie = ({config}) => <D3plusPie config={config} />;
+
+export const argTypes = {
+
+    /**
+     * Filters out unused argTypes from the Viz primitive and
+     * overrides any defaults that have been changed in Treemap
+     */
+    ...Object.keys(Viz.argTypes)
+      .filter(k => !k.match(/^(discrete|shape|zoom.*)$/))
+      .reduce((obj, k) => (obj[k] = Viz.argTypes[k], obj), {}),
+  
+    /**
+     * Pie-specific methods
+     */
+     innerRadius: {
+      type: {
+        summary: "function | number"
+      },
+      table: {
+        defaultValue: {
+          summary: "function | number",
+          detail: "0"
+        }
+      },
+      description: `The function or value to be used as inner radius in the Pie.`
+    },
```

Comment:
this should just be `summary: 0`. I'd like to only use the `detail` pop-up for long snippets of code. If we can display the value in a few characters, we should just display it as the summary.

please also add `defaultValue: 0, control: {type: "number"}` here so we can get a working UI 💪🏻 

description: The pixel value, or function that returns a pixel value, that is used as the inner radius of the Pie (creating a Donut).